io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+      - --with-avif=/app
+      - --with-png=/app
+      - --with-tiff=/app
+      - --with-webp=/app
+      - --without-x
+      - --without-xpm
+    build-options:
+      env:
+        CFLAGS: "-I/app/include"
+        LDFLAGS: "-L/app/lib"
+    build-commands:
+      - ./configure --prefix=/app --with-fontconfig=/app --with-freetype=/app --with-jpeg=/app --with-avif=/app --with-png=/app --with-tiff=/app --with-webp=/app --without-x --without-xpm
+      - make
+      - make install
+
+  - name: raster3d
+    sources:
+      - type: archive
+        url: http://www.bmsc.washington.edu/raster3d/Raster3D_3.0-7.tar.gz
+        sha256: f566b499fee341db3a95229672c6afdbdb69da7faabdbe34f6e0d332d766160c
+    buildsystem: simple
+    build-commands:
+      - |
+        export CFLAGS="-I${FLATPAK_DEST}/include"
+        export LDFLAGS="-L${FLATPAK_DEST}/lib"
+        export PATH="${FLATPAK_DEST}/bin:$PATH"
+        sed -i 's|prefix  = /usr/local|prefix  = /app|' Makefile.template
+        sed -i 's|INCDIRS  =	-I/usr/include -I/usr/local/include|INCDIRS  = -I/app/include|' Makefile.template
+        sed -i 's|LIBDIRS  =	-L/usr/local/lib|LIBDIRS  = -L/app/lib|' Makefile.template
+        sed -i 's|mandir  = $(prefix)/man/manl|mandir  = /app/man/manl|' Makefile.template
+        make linux
+        make all
+        make install
+
+  - name: ssm
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/dependencies/ssm-1.4.tar.gz
+        sha256: 56e7e64ed86d7d9ec59500fd34f26f881bdb9d541916d9a817c3bfb8cf0e9508
+    buildsystem: autotools
+    config-opts:
+      - --enable-ccp4
+      - --enable-shared
+      - --disable-static
+    build-options:
+      cflags: "-fPIC"
+    build-commands:
+      - ./configure --prefix=/app --enable-ccp4 --enable-shared --disable-static
+      - make
+      - make install
+
+  - name: boost
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --with-icu=/app --prefix=/app --libdir=/app/lib
+      - ./b2 install --prefix=/app --libdir=/app/lib -d2 -j$(nproc) threading=multi link=shared,static
+    config-opts:
+      - --without-libraries=[python, mpi]
+    build-options:
+      env:
+        CXXFLAGS: "-std=c++14"
+    cleanup:
+      - /app/share/boost
+
+  - deps/pypi-dependencies.json
+
+  - name: gobject-introspection
+    buildsystem: meson
+    config-opts:
+      - -Dpython=/usr/bin/python3
+    builddir: true
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
+        sha256: 0f5a4c1908424bf26bc41e9361168c363685080fbdb87a196c891c8401ca2f09
+    post-install:
+      - ln -sf /app/libexec/bin/* /app/bin/
+
+  - name: boost-python3
```

Comment:
why isn't this part of the boost module above?

Reply:
I'm trying to combine the two modules into one. I'll commit after the build passes locally.

Reply:
I couldn't, so have left the modules as they were.

Reply:
@pemsley 

What do you think about this?

Have you ever incorporated boost-python3 to boost build?

If you could, can you tell how to do it?

Reply:
I did.